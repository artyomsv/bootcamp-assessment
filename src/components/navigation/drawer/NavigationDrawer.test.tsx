import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import NavigationDrawer from './NavigationDrawer.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(<NavigationDrawer checked={true} navigateTo={mockFn}/>);
    expect(result).toMatchSnapshot();
  });
});
