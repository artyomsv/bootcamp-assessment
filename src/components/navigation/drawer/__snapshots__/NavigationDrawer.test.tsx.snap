// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<NavigationDrawer
  checked={true}
  classes={
    Object {
      "drawerPaper": "NavigationDrawer-drawerPaper-1",
    }
  }
/>
`;
